### ISSUE TITLE
Broken imports in 0.8.0

### ISSUE THREAD

**Issue by elibixby** [reanahub/reana-client#591]
After installing in a clean virtualenv I get this result:

```
Package                Version           
---------------------- ------------------
adage                  0.10.3            
amqp                   5.0.9             
appdirs                1.4.4             
argcomplete            2.0.0             
attrs                  21.4.0            
bagit                  1.8.1             
bravado                10.3.2            
bravado-core           5.17.0            
CacheControl           0.12.10           
certifi                2021.10.8         
charset-normalizer     2.0.12            
checksumdir            1.1.9             
click                  8.0.3             
coloredlogs            15.0.1            
ConfigArgParse         1.5.3             
connection-pool        0.0.3             
cwltool                3.1.20210628163208
datrie                 0.8.2             
decorator              5.1.1             
defusedxml             0.7.1             
docutils               0.18.1            
et-xmlfile             1.1.0             
filelock               3.4.2             
fs                     2.4.15            
gitdb                  4.0.9             
GitPython              3.1.26            
glob2                  0.7               
humanfriendly          10.0              
idna                   3.3               
ipython-genutils       0.2.0             
isodate                0.6.1             
jq                     1.2.2             
jsonpath-rw            1.4.0             
jsonpointer            2.2               
jsonref                0.2               
jsonschema             3.2.0             
jupyter-core           4.9.1             
kombu                  5.2.3             
lockfile               0.12.2            
lxml                   4.7.1             
mistune                0.8.4             
mock                   3.0.5             
monotonic              1.6               
msgpack                1.0.3             
msgpack-python         0.5.6             
mypy-extensions        0.4.3             
nbformat               5.1.3             
networkx               2.6.3             
odfpy                  1.4.1             
openpyxl               3.0.9             
packtivity             0.14.24           
pip                    20.0.2            
pkg-resources          0.0.0             
ply                    3.11              
prov                   1.5.1             
psutil                 5.9.0             
PuLP                   2.6.0             
pydot                  1.4.2             
pyparsing              3.0.7             
pyrsistent             0.18.1            
python-dateutil        2.8.2             
pytz                   2021.3            
PyYAML                 5.4.1             
ratelimiter            1.2.0.post0       
rdflib                 5.0.0             
reana-client           0.8.0             
reana-commons          0.8.4             
requests               2.27.1            
rfc3987                1.3.8             
ruamel.yaml            0.17.10           
ruamel.yaml.clib       0.2.6             
schema-salad           8.2.20220204150214
setuptools             44.0.0            
shellescape            3.8.1             
simplejson             3.17.6            
six                    1.16.0            
smart-open             5.2.1             
smmap                  5.0.0             
snakemake              6.8.0             
stopit                 1.1.2             
strict-rfc3339         0.7               
swagger-spec-validator 2.7.4             
tablib                 0.12.1            
tabulate               0.8.9             
toposort               1.7               
traitlets              5.1.1             
typing-extensions      4.1.1             
unicodecsv             0.14.1            
urllib3                1.26.8            
vine                   5.0.0             
webcolors              1.11.1            
Werkzeug               2.0.3             
wheel                  0.34.2            
wrapt                  1.13.3            
xlrd                   2.0.1             
xlwt                   1.3.0             
yadage                 0.20.1            
yadage-schemas         0.10.7  
```

Running `reana-client ping`
```
Traceback (most recent call last):
  File "/home/eli/.virtualenvs/reana/bin/reana-client", line 5, in <module>
    from reana_client.cli import cli
  File "/home/eli/.virtualenvs/reana/lib/python3.8/site-packages/reana_client/cli/__init__.py", line 16, in <module>
    from reana_client.cli import workflow, files, ping, secrets, quotas
  File "/home/eli/.virtualenvs/reana/lib/python3.8/site-packages/reana_client/cli/workflow.py", line 24, in <module>
    from reana_client.cli.files import get_files, upload_files
  File "/home/eli/.virtualenvs/reana/lib/python3.8/site-packages/reana_client/cli/files.py", line 21, in <module>
    from reana_client.cli.utils import (
  File "/home/eli/.virtualenvs/reana/lib/python3.8/site-packages/reana_client/cli/utils.py", line 25, in <module>
    from reana_client.utils import workflow_uuid_or_name
  File "/home/eli/.virtualenvs/reana/lib/python3.8/site-packages/reana_client/utils.py", line 30, in <module>
    from reana_commons.workspaces import validate_workspace
ModuleNotFoundError: No module named 'reana_commons.workspaces'
(reana) eli@eli-laptop:~/cradle/gitlab/infrastructure/reana$ reana-client ping
Traceback (most recent call last):
  File "/home/eli/.virtualenvs/reana/bin/reana-client", line 5, in <module>
    from reana_client.cli import cli
  File "/home/eli/.virtualenvs/reana/lib/python3.8/site-packages/reana_client/cli/__init__.py", line 16, in <module>
    from reana_client.cli import workflow, files, ping, secrets, quotas
  File "/home/eli/.virtualenvs/reana/lib/python3.8/site-packages/reana_client/cli/workflow.py", line 24, in <module>
    from reana_client.cli.files import get_files, upload_files
  File "/home/eli/.virtualenvs/reana/lib/python3.8/site-packages/reana_client/cli/files.py", line 21, in <module>
    from reana_client.cli.utils import (
  File "/home/eli/.virtualenvs/reana/lib/python3.8/site-packages/reana_client/cli/utils.py", line 25, in <module>
    from reana_client.utils import workflow_uuid_or_name
  File "/home/eli/.virtualenvs/reana/lib/python3.8/site-packages/reana_client/utils.py", line 30, in <module>
    from reana_commons.workspaces import validate_workspace
ModuleNotFoundError: No module named 'reana_commons.workspaces'
```

**Comment by mvidalgarcia** [comment]
Hi @elibixby, sorry we're in the middle of a release and about to release [`reana-client 0.8.1`](https://github.com/reanahub/reana-client/pull/589). You can install `reana-client 0.8.1a2` (`pip install reana-client --pre --upgrade` in your virtualenv) in the meantime.

Sorry for the inconvenience, we expect to release `reana-client 0.8.1` in the next few days.

**Comment by tiborsimko** [comment]
@elibixby Alternatively, if you are using REANA 0.8.0 on the cluster side, then the fastest fix is to downgrade the `reana-commons` package:

```console
$ pip install reana-commons==0.8.0 --upgrade
```

After which your `reana-client` 0.8.0 will work well.

**Comment by elibixby** [comment]
Thanks `0.8.1a2` fixed.

**Comment by mvidalgarcia** [comment]
@elibixby we've just released 0.8.1, you can upgrade to the latest version if you like.

FYI, we've also released a new REANA Helm chart, in case you're using a local cluster, you can check the details here: https://github.com/reanahub/reana/releases/tag/0.8.1
